pkg/Microsoft.ML.StaticPipe/Microsoft.ML.StaticPipe.nupkgproj

```diff
@@ -9,6 +9,7 @@
     <ProjectReference Include="../Microsoft.ML/Microsoft.ML.nupkgproj" />
     <ProjectReference Include="../Microsoft.ML.ImageAnalytics/Microsoft.ML.ImageAnalytics.nupkgproj" />
     <ProjectReference Include="../Microsoft.ML.Recommender/Microsoft.ML.Recommender.nupkgproj" />
+    <ProjectReference Include="../Microsoft.ML/Microsoft.ML.FastTree.nupkgproj" />
```

Comment:
This and the above are bad links.

`../Microsoft.ML/Microsoft.ML.FastTree.nupkgproj`

should be

`../Microsoft.ML.FastTree/Microsoft.ML.FastTree.nupkgproj`

Reply:
Sorry, clearly not alert today.